capabilities.json

```diff
@@ -63,6 +63,12 @@
             "displayName": "Extra information",
             "displayNameKey": "Role_ExtraInformation",
             "kind": "Grouping"
+        },
+        {
+            "name": "Tooltips",
+            "displayName": "Tooltips",
+            "displayNameKey": "Role_Tooltips",
```

Comment:
`Role_Tooltips` must be included to [resources.resjson](https://github.com/Microsoft/powerbi-visuals-gantt/blob/master/stringResources/en-US/resources.resjson) as well

Reply:
added